pkg/auth/local_authenticator.go

```diff
@@ -48,34 +48,34 @@ func (a *LocalAuthenticator) AuthAgentAuth(token string) (interface{}, error) {
 	t, err := validateToken(token, a.publicKey)
 	if err != nil {
 		a.log.WithError(err).Error("failed to validate token")
-		return nil, err
+		return nil, common.NewInfraError(401, err)
 	}
 	claims, ok := t.Claims.(jwt.MapClaims)
 	if !ok {
 		err := errors.Errorf("failed to parse JWT token claims")
 		a.log.Error(err)
-		return nil, err
+		return nil, common.NewInfraError(401, err)
```

Comment:
Why infra error? Shouldn't it be api error?

Reply:
Nope, every 401 is `infra_error` in the swagger.

Reply:
Correct, 401 and 403 are infra error